runtime_builders/template_builder.py

```diff
@@ -90,8 +94,13 @@ def _resolve_tags(config_file):
 
             for step in config.get('steps'):
                 image = step.get('name')
-                templated_step = _resolve_tag(image)
-                step['name'] = templated_step
+                step['name'] = _resolve_tag(image)
+                args = step.get('args', [])
+                for i in range(0, len(args)):
+                    arg = args[i]
+                    m = regex.search(IMAGE_REGEX, arg)
```

Comment:
in `_resolve_tag()`, I check to see if there's a tag (or digest) on the image, and log an error if not. so I'd rather just let the image pass through the regex and then do the check there if that's alright.